### PR TITLE
Devops 1574 unified release did not publish arm

### PR DESCRIPTION
**Pull request by Eeebru**
- Use skopeo to pull multi-arch image from ACR and push to docker hub

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -257,39 +257,22 @@ jobs:
...
       - name: Push version and latest image
         if: ${{ github.event.inputs.release_type != 'Dry Run' }}
         env:
           DOCKER_CONTENT_TRUST: 1
           DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE: ${{ steps.setup-dct.outputs.dct-delegate-repo-passphrase }}
         run: |
-          docker push bitwarden/self-host:$_RELEASE_VERSION
-          # docker push bitwarden/self-host:latest # TODO: uncomment this line after GA
+          ./skopeo --version
+          ./skopeo login $_AZ_REGISTRY -u 00000000-0000-0000-0000-000000000000 -p $(az acr login --expose-token --name bitwardenprod | jq -r .accessToken)
```
Comment (michalchecinski) [inline .github/workflows/release.yml:273]: Have you tested if that command leaves the access token in the logs?

Comment (Eeebru) [reply .github/workflows/release.yml:273]: That's a very good point raised, will check and revert

Comment (Eeebru) [reply .github/workflows/release.yml:273]: @michalchecinski I think the access token is not written to the logs, and I just added skopeo logout as well.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -257,47 +257,39 @@ jobs:
...
```
Comment (vgrassia) [inline .github/workflows/release.yml:273]:
```suggestion
          ./skopeo login $_AZ_REGISTRY -u 00000000-0000-0000-0000-000000000000 -p $(az acr login --expose-token --name ${_AZ_REGISTRY%.azurecr.io} | jq -r .accessToken)
```

**Review by vgrassia — CHANGES_REQUESTED** [review]

**Review by vgrassia — APPROVED** [review]

**Review by michalchecinski — APPROVED** [review]

**Pull request merged**